webextensions/api/action.json

```
@@ -118,6 +118,28 @@
               },
               "safari_ios": "mirror"
             }
+          },
+          "details_windowId_parameter": {
+            "__compat": {
+              "description": "<code>details.windowId</code> parameter",
+              "support": {
+                "chrome": {
+                  "version_added": false
+                },
+                "edge": "mirror",
+                "firefox": {
+                  "version_added": "109"
+                },
+                "firefox_android": {
+                  "version_added": false
+                },
```

```suggestion
                "firefox_android": "mirror",
```

Same for all other methods that take a `windowId` as `details` property.